### ISSUE TITLE
Allow chords

### ISSUE THREAD

**Issue by nathansobo** [atom/atom-keymap#17]
There's some discussion in https://github.com/atom/vim-mode/issues/221 about allowing   bindings based on multiple keys being smashed simultaneously. This is pretty tricky, because if we detected a `keydown` for a key involved in a chord, we'd need to wait for a `keyup` before we could be sure the chord wasn't being triggered. Not a super high priority right now, but worth implementing in the future.

**Comment by dnmiller** [comment]
I don't suppose there's a plan for this to make it into the 1.0 release, is there? It's really the last thing keeping me away from atom.

**Comment by maxbrunsfeld** [comment]
It sounds like many people following this issue are actually interested in key bindings for *sequences* like `k j` correctly inserting characters on partial match failures. Seems like this got a bit conflated with chords.

These kinds of sequences will be supported correctly in the next Atom release. Chords, where multiple printable character keys are hit simultaneously, are still not supported.

**Comment by BrainSlugs83** [comment]
+1 for sequence chords. (i.e. "ctrl-d, e", where you hold down "ctrl", and then type "d", and "e" sequentially, for example. -- it's a nice way to group related functionality under a root key.)

**Comment by jvivs** [comment]
+1 for @BrainSlugs83's described sequence chords

**Comment by sbrl** [comment]
Does this issue address sequence chords? e.g. `CTRL + R, CTRL + D`

**Comment by 50Wliu** [comment]
Maybe I'm misunderstanding what @BrainSlugs83, @jvivs, and @sbrl are asking for, but I believe that functionality is already available via the `ctrl-r ctrl-d` syntax, where you would first type `ctrl-r`, release, then type `ctrl-d`.

**Comment by sbrl** [comment]
@50Wliu Yep! That's what I'm after. In Visual Studio, they are called key chords I think. Is this issue for something different? I wonder if the documentation contains this.
